app/views/reporting/user_reports/index.html.erb

```diff
@@ -10,24 +10,45 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <h2 class='govuk-heading-m'>Latest complete monthly reports<h2>
+    <h2 class='govuk-heading-m'>Latest complete reports<h2>
```

Comment:
Are we using i18n for these or are you waiting until the copy / positioning is settled?

Reply:
yes, I imagine this page will change radically once designed :)